playground/backend/internal/environment/application.go

```diff
@@ -69,8 +74,13 @@ func NewCacheEnvs(cacheType, cacheAddress string, cacheExpirationTime time.Durat
 
 //ApplicationEnvs contains all environment variables that needed to run backend processes
 type ApplicationEnvs struct {
-	workingDir             string
-	cacheEnvs              *CacheEnvs
+	// workingDir is a root working directory of application
+	workingDir string
+
+	// cacheEnvs contains environments for cache
```

Comment:
why is it that we can have multiple cacheEnvs?

Reply:
No, we can have only one `cacheEnvs` which contains several `environment variables` (not several `environments`). Updated this comment.